hack/remove-calico-policy/calico-remove-iptables.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: calico-remove-iptables
+  namespace: kube-system
+  labels:
+    k8s-app: calico-remove-iptables
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: calico-remove-iptables
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/tolerations: |
```

Comment:
Let's just support this on Kubernetes v1.6+, in which case the tolerations should use the `tolerations` field rather than the now deprecated / unsupported annotation.